hydrocron/api/controllers/timeseries.py

```diff
@@ -125,23 +121,25 @@ def format_json(feature_lower, results: Generator, feature_id, fields):  # noqa:
 
 
 def format_csv(feature_lower, results: Generator, feature_id, fields):  # noqa: E501 # pylint: disable=W0613
-    """
-
-    Parameters
-    ----------
-    feature_lower
-    results
-    feature_id
-    fields
+    """ Format the results to the file format that the user selects (csv)
 
-    Returns
-    -------
+    :param feature_lower: Lowercase version of the type of feature
+    :type feature_lower: str
+    :param results: We pass the result of the query
+    :type results: dict
```

Comment:
type doesn't match the type in the function definition; is `results` a `Generator` or a `dict`?

Reply:
my mistake. fixed.